### PR TITLE
Update to Electron 21

### PR DESCRIPTION
**Pull request by gzdunek**
Closes https://github.com/gravitational/webapps.e/issues/369 
Closes https://github.com/gravitational/teleport-private/issues/175

Drone build https://drone.platform.teleport.sh/gravitational/teleport/17585 

I quickly tested the updated app and didn't see any issues.
  
I removed custom `node-gyp` resolution, it doesn't have any effect.

### CODE REVIEW

**Review by avatus — APPROVED** [review]
Worked for me, didn't notice anything

File: Dockerfile [hunk 1]
```diff
@@ -1,5 +1,5 @@
-FROM node:16.3-slim
-RUN apt-get update && apt-get install git g++ make python tree -y
+FROM node:16.18.1-slim
```
Comment (ravicious) [inline Dockerfile:1]: Do we need it to be pinned to a specific patch version or can we keep it pinned to a minor version like it was before?

Comment (gzdunek) [reply Dockerfile:1]: No, we don't have to. Changed to `16.18`.

**Review by ravicious — APPROVED** [review]
I did some tests on the dev version: window restoration, file transfer, terminal. Everything appears to be working fine.

**Pull request merged**